src/test/java/core/basesyntax/service/strategy/BalanceOperationStrategyTest.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class BalanceOperationStrategyTest {
+    private static Storage storage;
+    private static BalanceOperationStrategy strategy;
+
+    @BeforeAll
+    static void beforeAll() {
```

Comment:
you should clear your Storage efter each test that make changes to storage, to make your tests independent